src/mixins/filing-template-mixin.ts

```diff
@@ -113,6 +117,12 @@ export default class FilingTemplateMixin extends Vue {
     // Set Share Structure
     this.setShareClasses(draftFiling.incorporationApplication.shareClasses)
 
+    // Set Incorporation Agreement
+    this.setIncorporationAgreementStepData({
+      valid: !!draftFiling.incorporationApplication.incorporationAgreement.agreementType,
```

Comment:
I think we don't set 'valid' for the other data because each page validates itself when it is created. Can we leave this out here similary?

Reply:
Yes, I forgot to remove that. I will remove this